src/views/Product/Page.tsx

```diff
@@ -14,6 +13,8 @@ import GalleryCarousel from "./GalleryCarousel";
 import OtherProducts from "./Other";
 import { ProductDetails_product } from "./types/ProductDetails";
 
+import { ProductDescription as NewProductDescription } from "../../@next/components/molecules";
```

Comment:
May we just stay with ProductDescription? I assume it was just temporary change, am I right?

Reply:
There is a second component called ProductDescription(the old one) but I cant just throw it away and use new one instead as it does much more than the name suggests ;)